sklearn/metrics/pairwise.py

```diff
@@ -1015,19 +1015,35 @@ def paired_euclidean_distances(X, Y):
 
 
 def paired_manhattan_distances(X, Y):
-    """Compute the L1 distances between the vectors in X and Y.
+    """Compute the paired L1 distances between X and Y.
+
+    Distances are calculated between (X[0], Y[0]), (X[1], Y[1]), ...,
+    (X[n_samples], Y[n_samples]).
 
     Read more in the :ref:`User Guide <metrics>`.
 
     Parameters
     ----------
     X : array-like of shape (n_samples, n_features)
+        An array where each row is a sample and each column is a feature.
 
     Y : array-like of shape (n_samples, n_features)
+        An array where each row is a sample and each column is a feature.
 
     Returns
     -------
     distances : ndarray of shape (n_samples,)
+        L1 distances between the row vectors of `X` and the row vectors
+        of `Y`.
```

Comment:
Is this more exact?
```suggestion
        L1 paired distances between the rows vectors of `X`
        and the row vectors of `Y`.
```

Reply:
@jjerphan I think you have a typo in your suggestion that accidentally changed the first `row` to `rows`

Reply:
Nice catch! Corrected, thanks. :)